crates/cli/src/subcommands/publish.rs

```diff
@@ -32,6 +32,14 @@ pub fn cli() -> clap::Command {
                 .requires("name|address")
                 .help("When publishing to an existing address, first DESTROY all data associated with the module"),
         )
+        .arg(
+            Arg::new("build_options")
+                .long("build-options")
+                .alias("build-opts")
```

Comment:
optional nit: can we have a short for this as well? Maybe `-b` or `-o`?

Reply:
meh, nevermind. I think this syntax is weird:
```
spacetime generate -o '-S --debug'
```

`--build-opts` is not that intrusive